### PR TITLE
replace: replace banner img to SVG

### PR DESCRIPTION
**Pull request by deepakgithub11**

### CODE REVIEW

**Comment by deepakgithub11** [comment]
reponsive and scss file for index file is still pending for banner svg

File: src/app.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@ body {
 }
 
 .wpui-hero-Section {
-    background-image: url(img/banner.png);
+        background-color: transparent !important;
```
Comment (ajitbohra) [inline src/app.scss:7]: what do we need this ?

File: src/components/banner/index.js [hunk 2]
```diff
@@ -26,50 +27,523 @@ import './style.scss'
...
-                            Getting Started
-                        </Button>
-                        </Link>
-                    </HStack>
-                </VStack>
-            </CardBody>
-        </Card>
+        <>
+            <ZStack>
+                <svg className="svg"
```
Comment (ajitbohra) [inline src/components/banner/index.js:32]: extract this `SVG` into a separate component under `src/components` as `HeroBackground` or `BannerBackground`

**Review by ajitbohra — CHANGES_REQUESTED** [review]

**Comment by ajitbohra** [comment]
**_Issues:_**

We have the following issues:

**_Desktop_**
Screenshot 2024-06-28 at 7 02 30 PM

**_Mobile_**
Screenshot 2024-06-28 at 7 03 54 PM

**Comment by deepakgithub11** [comment]
banner is now responsive on phone

**Comment by ajitbohra** [comment]
Still banner is broken on both desktop and mobile!

**Comment by deepakgithub11** [comment]
banner svg is now responsive

**Pull request closed without merging**